dataflow/gemma/README.md

```diff
@@ -0,0 +1,152 @@
+# RunInference on Dataflow Streaming with Gemma
```

Comment:
maybe a walk through of how to test the pipeine while its running, so submit text with a prompt to A and then pull from B... etc,.

Reply:
See the "Send a prompt to the model" section